extension/package.json

```diff
@@ -204,6 +204,11 @@
         "title": "New Html Dialog",
         "category": "Axon Ivy"
       },
+      {
+        "command": "ivyProjects.addNewForm",
+        "title": "New Form",
```

Comment:
This is visible to the user right? -> I'd call it 'New Dialog Form' ... so that the Dialog concept is linkable to  the other dialog natures.

Reply:
yes it's visible, I would then streamline to "New Form Dialog"